building_sim_plugins/building_sim_ign/src/door.cpp

```diff
@@ -140,26 +31,27 @@ class IGNITION_GAZEBO_VISIBLE DoorPlugin
 private:
   rclcpp::Node::SharedPtr _ros_node;
   Model _model;
```

Comment:
It seems now this doesn't need to be a class variable anymore since it's only used in one function?